integration/test_collection_references.py

```diff
@@ -805,10 +806,7 @@ def test_empty_return_reference(collection_factory: CollectionFactory) -> None:
     )
 
 
-@pytest.mark.parametrize("to_uuid", [Reference.to(uuids=TO_UUID), TO_UUID, str(TO_UUID)])
-def test_refs_different_input_insert(
-    collection_factory: CollectionFactory, to_uuid: WeaviateReference
-) -> None:
+def test_refs_different_input_insert(collection_factory: CollectionFactory) -> None:
     to = collection_factory(name="To", vectorizer_config=Configure.Vectorizer.none())
     to_uuid = to.data.insert(properties={}, uuid=TO_UUID)
```

Comment:
this should not return a value, but the parametrized version should be used to test all possibilities